src/makie.jl

```diff
@@ -723,7 +714,7 @@ function histogram_or_bars_in_3d()
     cmap = get(colorschemes[cbarPal], ztmp)
     cmap2 = reshape(cmap, size(z))
     ztmp2 = abs.(z) ./ maximum(abs.(z)) .+ 0.15
-    fig = Figure(resolution = (1400,800), fontsize = 26)
+    fig = Figure(resolution = (1200,800), fontsize = 26)
     ax1 = Axis3(fig[1,1]; aspect = (1,1,1), elevation = π/6, perspectiveness = 0.5)
```

Comment:
```suggestion
    ax1 = Axis3(fig[1, 1]; aspect = (1, 1, 1), elevation = π / 6, perspectiveness=0.5)
```

Why only this pi is π?